app/scripts/modules/core/src/loadBalancer/LoadBalancerClusterContainer.tsx

```diff
@@ -44,7 +44,11 @@ export class LoadBalancerClusterContainer extends React.Component<ILoadBalancerC
         {showServerGroups && ServerGroups}
         {!showServerGroups &&
           showInstances && (
-            <LoadBalancerInstances serverGroups={loadBalancer.serverGroups} instances={loadBalancer.instances} />
+            <LoadBalancerInstances
```

Comment:
As far as I can see [`LoadBalancerInstance`](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/LoadBalancerInstance.java)s from Clouddriver's loadbalancers API do not include a cloudProvider field.  The cloudProvider field is used when the user clicks on an instance fronted by an LB ([here](https://github.com/spinnaker/deck/pull/5504/files#diff-d756080253c17bf1965714e31e06e50eR69)).  I'm not entirely sure how this works for other providers but threading the `cloudProvider` through from here down to `Instances.tsx` fixes the onClick handler.